pkg/ui/react-app/src/pages/graph/GraphControls.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { Component } from 'react';
+import { Button, ButtonGroup, Form, InputGroup, InputGroupAddon, Input } from 'reactstrap';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPlus, faMinus, faChartArea, faChartLine } from '@fortawesome/free-solid-svg-icons';
+
+import TimeInput from './TimeInput';
+import { parseRange, formatRange } from '../../utils';
+
+interface GraphControlsProps {
+  range: number;
```

Comment:
should one of these be `startTime` instead?

Reply:
This has been moved as-is from https://github.com/prometheus/prometheus/blob/d996ba20ec9c7f1808823a047ed9d5ce96be3d8f/web/ui/react-app/src/pages/graph/GraphControls.tsx#L11 which I think is a good thing since it makes this PR easier to review. Could we make a small PR upstream to fix it there? I think this PR can go either way with this.